packages/lisk-transactions/src/base_transaction.ts

```diff
@@ -134,33 +133,32 @@ export abstract class BaseTransaction {
 		transactions: ReadonlyArray<TransactionJSON>,
 	): ReadonlyArray<TransactionError>;
 
-	public constructor(rawTransaction: TransactionJSON) {
-		const valid = validator.validate(schemas.transaction, rawTransaction);
-		if (!valid) {
-			throw new TransactionMultiError(
-				'Invalid field types',
-				rawTransaction.id,
-				convertToTransactionError(rawTransaction.id || '', validator.errors),
-			);
+	// tslint:disable-next-line cyclomatic-complexity
+	public constructor(rawTransaction: unknown) {
+		const tx = (typeof rawTransaction === 'object' && rawTransaction !== null
+			? rawTransaction
+			: {}) as Partial<TransactionJSON>;
+		this.amount = new BigNum(tx.amount || '0');
```

Comment:
should we check typeof `amount` (and also `fee` below) before instantiating BigNum, as there is no longer a type check and it could be a non-number string which throws an error.